chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/loadbalancer/JLoadBalancer.java

```diff
@@ -66,7 +67,9 @@ public QueryPoolLoadBalancer build() {
             switch (loadBalancer) {
                 case ONE_BY_ONE:
                     if (seed == null) {
-                        return new OneByOneLoadBalancer();
+                        SimpleCircularLoadBalancer loadBalancer = new SimpleCircularLoadBalancer();
+                        loadBalancer.setPairSupplierFactory(new OneByOnePairSupplierFactory());
+                        return loadBalancer;
```

Comment:
I'm not sure. Code of *SupplierFactory differs from *LoadBalancer, so I'd prefer not to delete them.